### PR TITLE
Use the RPM OSTree module to install the payload

### PR DESCRIPTION
**Pull request by poncovka**
Use the DBus API of the RPM OSTree module in the RPM OSTree payload class.

### CODE REVIEW

File: pyanaconda/payload/rpmostreepayload.py [hunk 1]
```diff
@@ -54,149 +53,57 @@ def source_type(self):
...
-        # Following up on the "remote delete" earlier, we removed the remote from
-        # /ostree/repo/config.  But we want it in /etc, so re-add it to /etc/ostree/remotes.d,
-        # using the sysroot path.
-        #
-        # However, we ignore the case where the remote already exists, which occurs when the
-        # content itself provides the remote config file.
-        #
-        # Note here we use the deployment as sysroot, because it's that version of /etc that we
-        # want.
-
```
Comment (rvykydal) [inline pyanaconda/payload/rpmostreepayload.py:182]: Shouldn't we keep this information somewhere? Maybe in post_install_with_tasks.
Not sure if it is still accurate and comprehensible though.

**Review by rvykydal — APPROVED** [review]
Looks good to me.

**Review by VladimirSlavik — COMMENTED** [review]
Code LGTM. But I don't think the comments can just go like that.

File: pyanaconda/payload/rpmostreepayload.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/payload/rpmostreepayload.py:182]: The comment was moved to the `ChangeOSTreeRemoteTask` class.

Comment (VladimirSlavik) [reply pyanaconda/payload/rpmostreepayload.py:182]: Thanks!

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**